src/components/TodosError.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { useTodo } from '../providers/AppProvider';
+
+export const TodosError = () => {
+  const { errorTitle, setError } = useTodo();
+  const [hidden, setHidden] = useState(false);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setHidden(true);
+    }, 3000);
+
+    setTimeout(() => {
+      setError('');
+    }, 4000);
```

Comment:
I would refactor it, so that you only have one state responsible for showing error, which is `errorTitle` and `setError` 
I would delete `hidden` and `setHidden`.